app/models/ems_event.rb

```diff
@@ -338,6 +338,29 @@ def ems_refresh_target
     ext_management_system
   end
 
+  def src_container_replicator
```

Comment:
Hmm, this may be beyond my knowledge.
What I know is that the automate [event_action_policy][1] ends up passing the `target=src_...` parameter to [EmsEvent#get_target][2] which sends it to self as a method call.  (See [here][3] for more details.)
EmsEvent already has various type-specific methods like `dest_vm_refresh_target`...

[1]: https://github.com/ManageIQ/manageiq/blob/815d17f39be3f7977cd3de1f6fef544eb60e0b13/lib/miq_automation_engine/engine/miq_ae_builtin_method.rb#L70-L80
[2]: https://github.com/ManageIQ/manageiq/blob/master/app/models/ems_event.rb#L218-L233
[3]: https://github.com/ManageIQ/manageiq/pull/8815#issuecomment-220831496